src/GafferScene/MeshTessellate.cpp

```diff
@@ -59,6 +59,9 @@ MeshTessellate::MeshTessellate( const std::string &name )
 	addChild( new IntPlug( "divisions", Gaffer::Plug::In, 1, 1 ) );
 	addChild( new BoolPlug( "calculateNormals", Gaffer::Plug::In, false ) );
 	addChild( new StringPlug( "scheme", Gaffer::Plug::In, "" ) );
+	addChild( new StringPlug( "interpolateBoundary", Plug::In, "" ) );
+	addChild( new StringPlug( "faceVaryingLinearInterpolation", Plug::In, "" ) );
+	addChild( new StringPlug( "triangleSubdivisionRule", Plug::In, "" ) );
 	addChild( new BoolPlug( "tessellatePolygons", Gaffer::Plug::In, false ) );
```

Comment:
Perhaps `tessellatePolygons` would be better just underneath `scheme` where it was before? It is closely related to scheme (including being activated by it) and is less esoteric and more likely to be used than the new options.